### PR TITLE
Update hard-coded version to 1.3.0+dev

### PR DESCRIPTION
**Pull request by strk**
Was previously set to 1.1.0+dev

### CODE REVIEW

**Comment by strk** [comment]
Sorry this is for 1.3.0+dev (force-pushed).
Will follow PR for 1.2 branch too

**Comment by lunny** [comment]
How about 1.3.0-dev since it's target is 1.3.0 but isn't reach.

**Comment by strk** [comment]
I'm still confused about + vs -
Before the change we used +
What's the difference, semantically ?

Will 1.3.0-dev be older than 1.3.0 ?
Will 1.3.0+dev be newer than 1.3.0 ?

File: integrations/version_test.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 func TestVersion(t *testing.T) {
 	prepareTestEnv(t)
 
-	setting.AppVer = "1.1.0+dev"
+	setting.AppVer = "1.3.0+dev"
```
Comment (sapk) [inline integrations/version_test.go]: You forgot to update this like the main.go in the last commit

**Review by sapk — CHANGES_REQUESTED** [review]

**Comment by strk** [comment]
I realized that the test doens't need to match the hard-coded version.
I'm actually tempted to use "test" in there instead, to avoid the
confusion (and I guess that test could test at least two versions...)

**Comment by lunny** [comment]
It seems when we build via `TAGS="bindata" make generate build`, the version is still `Gitea version 1.1.0+534-g0307707ca built with: bindata`.

**Comment by strk** [comment]
That's because the Makefile rule generates version via `git describe`,
which does not read the hard-coded version number in source code.

If you want a better description there, you can add a 1.3 tag 
pointing to the first commit which got into master branch, ie:

  git tag 1.3 a915a09e4f8edc7734c9374ad9f9a51b39241ee3

**Comment by lunny** [comment]
So what's the meaning of the hard-code?

**Comment by strk** [comment]
It's for when you `go get` or `go build`, or build from sources
(outside of a git repository).

This PR does not change policy, just fixes its current application

**Comment by ethantkoenig** [comment]
LGTM

**Comment by strk** [comment]
@lunny so is this good to merge now ?

**Comment by Morlinest** [comment]
Shouldn't be version set only when you add tag? Now it should be `v1.2.0+dev` (or `v1.2.x+dev`) till next minor/major release.

Edit: Changed `-` to `+` as `-` means pre-release.

**Comment by lunny** [comment]
For master, I suggest always use `-` but `+`. It should be `v1.3.0-dev` means next release is v1.3.0

**Comment by daviian** [comment]
LGTM

**Pull request merged**